app/controllers/emails_controller.rb

```diff
@@ -54,15 +54,22 @@ def save_template
       flash.now[:alert] = t('.saving_failed')
     end
     @event = Event.find(params[:event_id])
-    @templates = EmailTemplate.with_status(get_status)
+    @templates = EmailTemplate.with_status(get_email_template_status)
 
     render :email
   end
 
-  def get_status
+  def get_email_template_status
     params[:status] ? params[:status].to_sym : :default
   end
 
+  def get_corresponding_application_letter_status
```

Comment:
Usually yes, but its a private controller helper. Calling this function alone doesnt make sense at all (and its not ment to be accessible from "outside". And I think that we never tested private helper methods (like e.g. event_params or such), so it seems inconsistent for me if we add a test for this. Instead, we test the public function that calls it.